src/System.Memory/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>PKV006</DiagnosticId>
+    <Target>.NETStandard,Version=v1.1</Target>
```

Comment:
@ViktorHofer I saw a similar file in one of the other migrated libraries. Since you didn't say anything about it, I assume the change is correct, right?

I do have a question: Why was APICompat complaining about netstandard1.1 here? Is it because netcoreapp2.1 is compatible with netstandard1.1?

Reply:
PKV006 means target framework dropped: https://github.com/dotnet/sdk/blob/bc9da06d9512a3db932bf638cab8f8e3f5b5aea9/src/Compatibility/ApiCompat/Microsoft.DotNet.PackageValidation/DiagnosticIds.cs#L19C48-L19C48

The latest package version (4.5.5) supports netstandard1.1:

<img width="447" alt="image" src="https://github.com/dotnet/maintenance-packages/assets/7412651/f9e59cd6-78db-4d7b-9621-ae3ed0f0cd46">

So PackageValidation is telling you that you are dropping support for .NET Standard 1.1.